packages/subgraph/schema.graphql

```diff
@@ -0,0 +1,301 @@
+# Values for the current status/stage of the QFI.
+enum Stage {
+  NotInitialized
+  WaitingForSignupsAndTopups
+  VotingPeriodOpen
+  WaitingForFinalization
+  Finalized
+  Cancelled
+}
+
+# Values for the RecipientRegistry requests.
+enum RequestType {
+  Registration
+  Removal
+}
+
+# QFI Core entity.
+type QFI @entity {
+    id: ID!
+
+    # Main fields.
+    coordinator: Coordinator
+    currentStage: Stage! # Enum.
+    voiceCreditFactor: BigInt!
+    nextGrantRoundId: BigInt!
+    contributorCount: BigInt!
+    isStateAqMerged: Boolean
+
+    # Inherited from MACI.
+    stateTreeDepth: Int
+    numSignUps: BigInt!
+    isInitialized: Boolean
+    signUpTimestamp: BigInt!
+    # Useful smart contracts addresses.
+    signUpGatekeeperAddress: Bytes!
+    initialVoiceCreditProxyAddress: Bytes!
+    vkRegistryAddress: Bytes
+    pollFactoryAddress: Bytes!
+    messageAqFactoryPollAddress: Bytes
+    stateAqAddress: Bytes
+    nativeERC20TokenAddress: Bytes!
+    messageAqFactoryGrantRoundAddress: Bytes
+    grantRoundFactoryAddress: Bytes!
+    pollProcessorAndTallyerAddress: Bytes
+
+    # Links.
+    recipientRegistry: RecipientRegistry
+    currentGrantRound: GrantRound
+
+    # Reverse Lookups.
+    grantRounds: [GrantRound!] @derivedFrom(field: "qfi")
+    # Specifically, those Polls are GrantRounds (polls[pollId] = g, where g is a GrantRound).
+    polls: [GrantRound!] @derivedFrom(field: "qfi")
```

Comment:
Apologies for missing the single Poll deployment without passing through a GrantRound. Additionally, I have also forgotten to handle the `DeployPoll()` event.

I was wondering if we could make a Poll entity to handle the core shared data (e.g., `maxValues`, `treeDepths`, etc.) between Poll/GrantRound. Idk if it could be better than using a boolean `isPool` on the GrantRound entity. With the Poll entity, we only need to populate one of it when creating a Poll; thus creating a GrantRound needs Poll+GrantRound entities linked together. 

Idk if this approach is more computational/logically efficient when querying, I'd like to hear from you about that. Thanks